### PR TITLE
Feature/432/unit test filter cell

### PR DESCRIPTION
**Pull request by FryDay969**
write unit tests for components/enchanced-table/filter-cell

### CODE REVIEW

File: src/components/search-input/SearchInput.jsx [hunk 1]
```diff
@@ -45,6 +46,7 @@ const SearchInput = ({ search, setSearch }) => {
       sx={ styles.input }
       value={ searchInput }
       variant='standard'
+
```
Comment (MickhaelSh) [inline src/components/search-input/SearchInput.jsx:49]: remove empty lines, you have added

File: tests/unit/components/enhanced-table/filter-row/filter-cell/FilterCell.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { vi } from 'vitest'
+import { screen, render, fireEvent } from '@testing-library/react'
+import FilterCell from '../../../../../../src/components/enhanced-table/filter-row/filter-cell/FilterCell'
```
Comment (MickhaelSh) [inline tests/unit/components/enhanced-table/filter-row/filter-cell/FilterCell.spec.jsx:3]: use ~ for imports

File: tests/unit/components/enhanced-table/filter-row/filter-cell/FilterCell.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+              const searchIcon = screen.getByTestId('FilterAltIcon')
+        
+              expect(searchIcon).toBeInTheDocument()
+
+              const clearIcon = screen.getByTestId('ClearIcon')
+    
+          expect(clearIcon).toBeInTheDocument()
+            })
+    
+  })
```
Comment (MickhaelSh) [inline tests/unit/components/enhanced-table/filter-row/filter-cell/FilterCell.spec.jsx:66]: formating in this file

File: src/components/search-input/SearchInput.jsx (hunk 1 above)
Comment (FryDay969) [reply src/components/search-input/SearchInput.jsx]: Fixed

File: tests/unit/components/enhanced-table/filter-row/filter-cell/FilterCell.spec.jsx (hunk 2 above)
Comment (FryDay969) [reply tests/unit/components/enhanced-table/filter-row/filter-cell/FilterCell.spec.jsx]: Fixed

**Review by MickhaelSh — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Pull request merged**